neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/HttpNeo4jDialect.java

```diff
@@ -558,6 +558,22 @@ public void forEachTuple(ModelConsumer consumer, TupleTypeContext tupleTypeConte
 		}
 	}
 
+	@Override
+	public int executeBackendUpdateQuery(BackendQuery<String> query, QueryParameters queryParameters, TupleContext tupleContext) {
+		Map<String, Object> parameters = getParameters( queryParameters );
+		String nativeQuery = buildNativeQuery( query, queryParameters );
+
+		Statement statement = new Statement( nativeQuery, parameters );
+		statement.setResultDataContents( Arrays.asList( Statement.AS_ROW ) );
+
+		Statements statements = new Statements();
+		statements.addStatement( statement );
+
+		Long txId = transactionId( tupleContext.getTransactionContext() );
+		StatementsResponse response = client.executeQueriesInOpenTransaction( txId, statements );
+		return 0;
```

Comment:
I wonder if -1 would be better. It better states that it is an unknown value. WDYT?